orientdb/src/main/java/org/hibernate/datastore/ogm/orientdb/dialect/impl/OrientDBAssociationQueries.java

```diff
@@ -6,17 +6,32 @@
  */
 package org.hibernate.datastore.ogm.orientdb.dialect.impl;
 
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import org.hibernate.datastore.ogm.orientdb.constant.OrientDBConstant;
+import org.hibernate.datastore.ogm.orientdb.logging.impl.Log;
+import org.hibernate.datastore.ogm.orientdb.logging.impl.LoggerFactory;
+import org.hibernate.datastore.ogm.orientdb.utils.EntityKeyUtil;
+import org.hibernate.ogm.model.key.spi.AssociationKey;
 import org.hibernate.ogm.model.key.spi.AssociationKeyMetadata;
+import org.hibernate.ogm.model.key.spi.EntityKey;
 import org.hibernate.ogm.model.key.spi.EntityKeyMetadata;
-import org.hibernate.ogm.util.impl.Log;
-import org.hibernate.ogm.util.impl.LoggerFactory;
+import org.hibernate.ogm.model.key.spi.RowKey;
 
 /**
- * @author chernolyassv
+ * @author Sergey Chernolyas (sergey.chernolyas@gmail.com)
  */
 public class OrientDBAssociationQueries extends QueriesBase {
 
-	private static final Log log = LoggerFactory.make();
+	private static final Log log = LoggerFactory.getLogger();
 
```

Comment:
My bad, it seems `getLogger()` it seems the one in core is called `make()` while all the other in the dialects are called `getLogger()`